hack/build/bazel-docker.sh

```diff
@@ -81,7 +84,7 @@ printf "\n"
 rsynch_fail_count=0
 
 _rsync() {
-    rsync -al "$@"
+    rsync -al --no-owner "$@"
```

Comment:
no need to do this and `git config --global --add safe.directory`. I say we hold off until there is a real need

Reply:
The invocation of `git config` can be optimized  by running it only after the volume is created.  But I think as-is is fine.

I don't see why the `--no-owner` state is needed or desired now or in the future.  Please explain why it is.  My fear is that it may have unintended consequences and why take on that risk?